### ISSUE TITLE
Error reported when changing volume

### ISSUE THREAD

**Issue by Paul-Reed** [Hypnos3/node-red-contrib-cast#23]
My user case is that I want to start each day with a preset volume level for my Google Home. This is necessary if I've been listening to music at a low level at bedtime, or someone else has lowered the volume.
So I have set a simple flow which runs every morning at 8.30am, injects msg.volume = 60 into the cast node.

![home](https://user-images.githubusercontent.com/973580/48666236-c16e1f80-eab5-11e8-88f2-2b85012480fe.PNG)
You can see the error below the node, and also, in the debug panel I get;
`"Exception occured on load media:Cannot read property 'transportId' of undefined"` ...although the volume level does actually successfully change in the device.
I guess this is linked to issue https://github.com/Hypnos3/node-red-contrib-cast/issues/22

I'm assuming that it's because I'm not injecting a media file url, because if I add a silent '1 second' mp3 file url, the error does not appear, but that's not really a solution because the 'silent mp3' is always preceded in the Home device by a introductory beep, so it's not really silent!

```
[{"id":"c054f1da.962f","type":"cast-to-client","z":"c53060.842a0fa","name":"Google Home","url":"","contentType":"","message":"","language":"en","ip":"","port":"","volume":"","x":480,"y":500,"wires":[[]],"icon":"node-red-contrib-cast/home.png"},{"id":"89dd5292.8dd33","type":"change","z":"c53060.842a0fa","name":"Lounge","rules":[{"t":"set","p":"ip","pt":"msg","to":"192.168.1.5","tot":"str"},{"t":"set","p":"volume","pt":"msg","to":"50","tot":"num"}],"action":"","property":"","from":"","to":"","reg":false,"x":290,"y":500,"wires":[["c054f1da.962f"]]},{"id":"46fcb48e.76bcfc","type":"inject","z":"c53060.842a0fa","name":"","topic":"","payload":"","payloadType":"str","repeat":"","crontab":"30 08 * * *","once":false,"onceDelay":0.1,"x":120,"y":500,"wires":[["89dd5292.8dd33"]]}]
```

**Comment by Hypnos3** [comment]
It is the same issue as #22, I will search for a solution.

**Comment by Hypnos3** [comment]
I had released Version 0.2.0, which hopefully should fix that Bug.

**Comment by Paul-Reed** [comment]
Nice work Stephen, working great now. No errors!
Fixed by https://github.com/Hypnos3/node-red-contrib-cast/commit/754edfa81348fe56c3b1339c67dcaef899a0f1f5
